docs/examples/quick-start-with-terraform.md

```diff
@@ -0,0 +1,302 @@
+# Quick Start with Terraform
+
+This tutorial aims at presenting how to set up an environment with Liqo installed via Terraform and explore its most notable capabilities.
```

Comment:
Not clear who "notable" refers to? Is it Terraform that has notable capabilities? Or Liqo?
Please clarify by rephrasing.

Reply:
We will try to clarify the aims by rephrasing in next commit.